tests/integration/functions/define_function_test.ts

```diff
@@ -1,6 +1,5 @@
-import { DefineFunction } from "./mod.ts";
-import Schema from "../schema/mod.ts";
-import { assertEquals, assertStrictEquals } from "../dev_deps.ts";
+import { assertEquals, assertStrictEquals } from "../../../src/dev_deps.ts";
```

Comment:
Why was this moved to be an integration test? AFAIK this file is performing unit tests on `DefineFunction` which is really testing the underlying `SlackFunctionDefinition` class logic.

@filmaj can correct me but my understanding is our integration files are files that are intentionally testing two separate imports together

My two cents is to just test `DefineFunction` and `DefineConnector`, which will execute the corresponding function definition

Reply:
I decided to move `define_function_test.ts` here since it seem contain more then just "pure unite tests", this file seems to validate the full behavior of `DefineFunction`

Example: 

- "DefineFunction with required params"
- "DefineFunction using an OAuth2 property allows require_end_user_auth"

Since theses tests cover all the new changes (`FunctionDefinition` abstract class + SlackFunctionDefinition) I thought they resembled more integration tests rather then "pure unite" tests

Reply:
Both of those examples stem from the underlying implementation thoug, rather than coming from `DefineFunction` itself. I'd think we should just move them to the corresponding file?

Reply:
I've moved these test back to the unit test file